db/seeds.rb

```diff
@@ -13,32 +13,32 @@
 manu = User.create(name: 'Manuel Pérez', email: 'manu@paraffin.com', password: 'password')
 
 fin = Curriculum.create(name: 'FIN: Fullstack Developer')
-docker = LearningUnit.create(name: 'Docker')
-git = LearningUnit.create(name: 'Git - control de versiones')
-ruby = LearningUnit.create(name: 'Ruby')
-rails = LearningUnit.create(name: 'Rails')
-js = LearningUnit.create(name: 'JavaScript')
-react = LearningUnit.create(name: 'React')
-nextjs = LearningUnit.create(name: 'NextJS')
+docker = LearningUnit.create(name: 'Docker', description: 'Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.')
```

Comment:
Even if the `image_url` is nullable, we should have a dummy image_url, so we have at least one image to test.

Reply:
Done